strawberry/types/type_resolver.py

```diff
@@ -260,22 +313,25 @@ def _get_fields(cls: Type) -> List[FieldDefinition]:
                 default_value=getattr(cls, field.name, undefined),
```

Comment:
Why not just use `dataclasses.MISSING`?

Reply:
I didn't think of that, we can change it 😊

Reply:
Will address in another PR

Reply:
Created issue #439